.devcontainer/Dockerfile

```diff
@@ -20,6 +20,11 @@ RUN apt-get update \
     && apt-get update \
     && apt-get install -y docker-ce-cli \
     #
+    # Insall Azure CLI

```

Comment:
```suggestion
    # Install Azure CLI

```

Reply:
Great catch. All the more reason for a spell checker lolol